src/triggered/CommunicationMapping.ts

```diff
@@ -0,0 +1,125 @@
+import { LitElement } from 'lit-element';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
import { LitElement } from "lit-element";
```